src/importData.jl

```diff
@@ -54,6 +54,141 @@ function import_combination(filename::String)
     return gs[:, :, 2:25], gs[:, :, 27:50], perc, total
 end
 
+""" This function imports the new round of data and filters. """
+function import_data()
+    # 96 conditions(not separating replicates) with 191 data points,
+    # and 1158 conditions(not separating conditions) with 192 data points.
+    # 18336 / 191 = 96
+    # 222336 / 193 = 1152
+    # so we have a total of 1254 conditions
+    basePath = joinpath(dirname(pathof(DrugResponseModel)), "..", "data")
+
+    d = zeros(193, 1248, 2) # time points x conditions x G1% and total
+
+    dataname = "AU565_round2and3_model_subset_level2.csv"
+    data = readdlm(joinpath(basePath, dataname), ',')
+
+    conditions = Vector{String}()
+
+    for i = 1:96 # first set that have 191 data points
```

Comment:
All this looping should be a table [reshape](https://dataframes.juliadata.org/stable/man/reshaping_and_pivoting/).